### ISSUE TITLE
Using as Bridge with 3rd party HaLow AP, Scan results in ERROR

### ISSUE THREAD

**Issue by jrigling** [Xinyuan-LilyGO/T-Halow#7]
I intend to use the T-Halow as a network Bridge (HaLow to Ethernet) for connecting non-HaLow devices to a HaLow network. This is like Mode 2 in the documentation, except the AP is NOT a T-HaLow device. The AP uses a Newracom chipset. The AP has been tested with another independent device, using a MorseMicro chipset. 

This strategy will not use the AT+PAIR= behavior since it will be using an SSID. 

The ESP32 is not used in this configuration. 

The AP is configured with 
BSS_BW=4MHz
Frequency=910MHz
SSID=Wi-Fi
WIFI_PSK=none

The logs below are from firmware `huge-ic-ah_v1.6.4.3-29063_2024.8.16_.bin` 

```
LMAC STATUS:
local: 4a:06:59:86:b2:b8 AID= 0
freq= 910.0 bw=4 chn=3 bgr=-60 iq=0:50:-1:0 dc=0:0(28:50) tx=*5 dly=72:0:0 sif:rsp=0:0(0)
chn:  910.0 
bgr:    -60 
buf: free=78 tq=0 sq=0 ac=0:0:0:0 agg=0:0:0:0 statq=0 rxq=0:0 skb=0:0 rxb=120K
irq: ac=9 bkn=0 bo(rts:frm)=0:0:0:0:9 to(rts:frm)=0:0 rx=45
tx : cnt=9 dly=0:0ms mcast(bw:mcs)=2:0 agg=1(5994,9:0) data=1KB(1kbps) dur=6ms cca=53 per=0% est_rate=1300kbps fail=0 drop=0
rx : cnt=20 bus=1ms cts_bm=0:0 pks=20 data=3KB(4kbps) dur= 7ms err(phy:fcs)=0:0 ecode=0x0 rssie= 0:0 cache_rxq= 0:0
dbg: dtmd=0:0:0 stamap=0x0 sleepmap=0x0 flag=0x0 rxdp=0 kerr=0 mic=0 lerr=0 kick=0 csc=0 rst=2 ovf=0 nob=0 tsnr=0 rssi=0 rxdut=0% txp=0 rxg=5    
cca: 4s st12= 41:0 mid1224= 50:44:0:0 ed1224= 50:18:0:0 ch_bz= 0
chip-temperature:44, vcc:3.26, vdd13b:1.31, vdd13c:1.31
sta_list: no sta
[2463184]send assoc request to ff:ff:ff:ff:ff:ff, ssid=Wi-Fi (freq:9100)
[2463734]send assoc request to ff:ff:ff:ff:ff:ff, ssid=Wi-Fi (freq:9100)
[2463761]send assoc request to ff:ff:ff:ff:ff:ff, ssid=Wi-Fi (freq:9100)
```

After setting the BSS_BW and SSID with AT commands, the logs start to include attempts to associate, but the address is `FF:FF:FF:FF:FF:FF`, which doesn't match the AP. Is this expected?

I also changed the channel list with `AT+CHAN_LIST=9100` and the PSK with `AT+KEYMGMT=NONE` to match the 3rd party AP configuration. 

I try to request `AT+SCAN_AP` so I can then `AT+BSSLIST?` and see if the AP is discovered. The `AT+SCAN_AP` returns `ERROR`

What should I do to understand why the AP SCAN results in an error?  Is this causing the STA to not find the AP?

**Comment by aliosa27** [comment]
the scan command seem to be broken on the AT firmware. I have also noticed that trying to associate to a newracom device fails, while newracom -> morse works. The halow stack seems to be a little different here. I think you might have to reach out directly to taixin.

**Comment by Humancell** [comment]
Yes ... adding to this ... I have tried to test extensively getting these Lilygo/Taixin devices working with certified 802.11ah Newracom and Morse Micro devices and they do NOT work.

I have reached out to Taixin numerous times asking for assistance, and have got no reply.

https://en.taixin-semi.com/Product?prouctSubClass=33

From what I can tell, their chipset or firmware are NOT 802.11ah compliant.  :-(

This one module is listed on the WiFI Alliance website as being certified ... but maybe not the one on this product?

Product Name: IEEE802.11ah Halow Wireless Module
Model Number: TXW8301
Total Variants: 1 (1 result)
Brand: Taixin Semiconductor Co., Ltd.
Category: Other
Last Certified Date: 2024-04-26
